src/test/rustdoc-ui/intra-doc/anchors.rs

```diff
@@ -43,3 +43,7 @@ pub fn enum_link() {}
 /// [u32#hello]
 //~^ ERROR `u32#hello` contains an anchor
 pub fn x() {}
+
+/// [prim@usize#x]
```

Comment:
Actually, why is this an error? Shouldn't this link to an anchor in a page like [primitive.usize.html](https://doc.rust-lang.org/std/primitive.usize.html)? I guess this also goes for the test case above (`[u32#hello]`). I think I would find a link to `usize::#examples-3` actually reasonable. What am I missing?

Reply:
This is https://github.com/rust-lang/rust/issues/83083, it's a bug but pretty hard to fix.